src/components/slider/slider.html

```diff
@@ -0,0 +1,14 @@
+<div class="bx--form-item" style="width: 600px;">
```

Comment:
Just to be clear: this inline style something the user should override if they want the slider to be a different width?